docs/explanations/introduction.md

```diff
@@ -0,0 +1,29 @@
+
+Morpho is an implementation of such a checkpointing federation, based on [OBFT](https://iohk.io/en/research/library/papers/ouroboros-bfta-simple-byzantine-fault-tolerant-consensus-protocol/) to provide consensus and byzantine fault tolerance among the federation members. It is built to interact with [Mantis](https://github.com/input-output-hk/mantis), an Ethereum Classic client that implements support for checkpoint blocks. This scheme has been proposed in [ECIP 1097](https://ecips.ethereumclassic.org/ECIPs/ecip-1097), which has been previously [discussed](https://www.youtube.com/watch?v=dentctKr880), and [demonstrated](https://www.youtube.com/watch?v=1JeL4pAjmXM). A formal specification of Morpho is available [here](https://github.com/input-output-hk/OBFT-Checkpointing-Specification/blob/master/formal-spec/formal-spec.pdf).
+
+## Topology
+
+The checkpointing federation consists of a fixed number of Morpho nodes (configured with [`NumCoreNodes`](../references/configuration.md#numcorenodes)), each of which is run by a trusted party. These Morpho nodes are connected to each other via TCP, striving for a fully connected network, meaning every node has a direct connection to every other node. The connections for this are handled by the [OBFT library](https://github.com/input-output-hk/ouroboros-network). The Morpho topology is specified with the file passed in [`TopologyFile`](../references/configuration.md#topologyfile), whose format is documented [here](../references/topology.md).
+
+In addition to the Morpho nodes, each host of the federation also runs a passive [Mantis](https://github.com/input-output-hk/mantis) node. This node allows the checkpointing federation to interact with the ETC network.
+
+In order for Morpho to know which proof-of-work blocks it can attempt to checkpoint, and to publish the created checkpoints, it issues [JSON RPC](https://www.jsonrpc.org/) requests to its corresponding Mantis node. These connections are local to the federation host and therefore not exposed to the internet. This RPC interface is documented [here](../references/rpc.md).
```

Comment:
In order for -> For
checkpoint, and -> checkpoint and
[here] -> [in the RPC page]